React/game/src/app/components/Screen/Game/Board/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+
+import Square from '../Square';
```

Comment:
you can use your brand new knowledge on `aliases` here, for example :D

Reply:
👍

Reply:
👍